bindgen generates rust codes that cannot be compiled with rust-1.79.0 on EPEL9
  https://github.com/rust-lang/rust-bindgen/issues/3052

Add Workaround for this build error on el9

error: extern block cannot be declared unsafe
 --> ../../third_party/crabbyavif/src/sys/dav1d-sys/src/../../../../../../out/Release/gen/third_party/crabbyavif/crabbyavif_dav1d_sys/dav1d_bindgen.rs:3:18605

Signed-off-by: Than Ngo <than@redhat.com>

diff -up chromium-136.0.7103.92/build/rust/run_bindgen.py.than chromium-136.0.7103.92/build/rust/run_bindgen.py
--- chromium-136.0.7103.92/build/rust/run_bindgen.py.than	2025-05-08 14:09:50.268246973 +0200
+++ chromium-136.0.7103.92/build/rust/run_bindgen.py	2025-05-08 14:46:22.421719175 +0200
@@ -68,7 +68,7 @@ def main():
     # TODO(danakj): We need to point bindgen to
     # //third_party/rust-toolchain/bin/rustfmt.
     genargs.append('--no-rustfmt-bindings')
-    genargs += ['--rust-target', 'nightly']
+    genargs += ['--rust-target', '1.79']
 
     if args.depfile:
       depfile = stack.enter_context(action_helpers.atomic_output(args.depfile))
